services/core-web/src/components/noticeOfWork/NOWProgressActions.js

```diff
@@ -93,7 +93,8 @@ export class NOWProgressActions extends Component {
   handleStartDelay = (values) => {
     const payload = {
       ...values,
-      start_date: new Date(this.props.noticeOfWork.last_updated_date),
+      // start_date: new Date(this.props.noticeOfWork.last_updated_date),
```

Comment:
Remove?

Reply:
@mmumby ?

Reply:
Ryan has some temporary changes in his branch, as you can see. 

This will also be resolved in my follow up PR.